airflow-code-editor/airflow_code_editor/flask_admin_view.py

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env python
 #
 #   Copyright 2019 Andrea Bonomi <andrea.bonomi@gmail.com>
+#   Modifications included by DSCOLabs engineering team
+#   University of San Francisco, CA
```

Comment:
super-nit, but these modification statements should probably all be "University of California, San Francisco"?